src/ui/Searchbox/Searchbox.ts

```diff
@@ -51,6 +53,12 @@ export class Searchbox extends Component {
      * Default value is `true`.
      */
     addSearchButton: ComponentOptions.buildBooleanOption({ defaultValue: true }),
+    /**
+     * The height in pixel that the Searchbox will be rendered with
```

Comment:
Suggestion:

> A custom height for the search box (in pixels).
>
> **Note:** Avoid setting this option if you want to modify the search box styling through custom CSS.
>
> **Minimum:** `25`

The note seems important to me since setting this option populates the `style` attribute of some HTML elements. What do you think?

Reply:
Agreed.
It should also explain what happens when the value is not set, how does the default gets set ?

Reply:
Good idea, here's a revised suggestion:

> A custom height for the search box (in pixels).
>
> **Note:** Avoid setting this option if you want to modify the search box styling through custom CSS.
>
> **Minimum:** `25`
>
> **Default behavior:** The most specific CSS rules targeting HTML elements generated by the `Searchbox` component are used (e.g., if you are only using the default `CoveoFullSearch.css` stylesheet with no further styling customizations, the search box height will be set to 50 pixels by default).